recipe/meta.yaml

```diff
@@ -17,16 +17,16 @@ source:
 
 build:
   skip: true  # [win and vc<14]
-  number: 3
+  number: 4
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}  # Needed because CMake project is not declared as C
     - cmake
     - ninja
-  host:  # [local_blas_impl == 'openblas']
-    - openblas-devel  # [local_blas_impl == 'openblas']
+  host:
+    - libblas  # [unix]
```

Comment:
Looks good. If there was a way to build both blas and noblas versions with the same recipe it would be even better ;)